static/app/components/modals/explore/saveQueryModal.tsx

```diff
@@ -42,14 +42,14 @@ function SaveQueryModal({
   Body,
   Footer,
   closeModal,
-  queries,
   saveQuery,
   name: initialName,
 }: Props) {
   const organization = useOrganization();
 
   const [name, setName] = useState(initialName ?? '');
   const [isSaving, setIsSaving] = useState(false);
+  const [starred, setStarred] = useState(true);
```

Comment:
We start with everything being starred? Just making sure, since it seems a bit unconventional

Reply:
Yeah I left the option to true by default. The user can toggle it off before saving the query if they want. I'll leave it like this for now, but if we can change easily if we get negative feedback